tutorials/meshing_of_fractures.ipynb

```diff
@@ -4,89 +4,153 @@
    "cell_type": "markdown",
```

Comment:
<p>Have you coordinated this revision with the group working on updating the tutorials?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I will have a chat with them today.

Reply:
The tutorial now points to [bc0288b](https://github.com/pmgbergen/porepy/commit/bc0288b50da7a4760aee69dc11d9498df95c81be). Since this will be properly reviewed as part of the tutorial updates, there is no need to review it now.